code/modules/admin/tabs/admin_tab.dm

```diff
@@ -421,6 +421,27 @@
 	remove_verb(src, admin_verbs_hideable)
 	add_verb(src, /client/proc/enable_admin_verbs)
 
+/client/proc/strip_all_in_view()
+	set name = "Strip All"
+	set category = "Admin.InView"
+	set hidden = TRUE
+
+	if(!admin_holder || !(admin_holder.rights & R_MOD))
+		to_chat(src, "Only administrators may use this command.")
+		return
+
+	if(alert("This will strip ALL mobs within your view range. Are you sure?",,"Yes","Cancel") == "Cancel")
```

Comment:
```suggestion
	if(tgui_alert(src, "This will strip ALL mobs within your view range. Are you sure?", "Confirmation", list("Yes", "Cancel")) == "Cancel")
```